.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+
+rvm:
+- 2.2
+- 2.1
+- 2.0.0
+- 1.9.3
+- 1.8.7
+- jruby-19mode
+- rbx-2
+
+sudo: false
+
+cache: bundler
+
+after_success:
```

Comment:
Sadly, no. I looked into that originally. The only `deploy` step for GitHub supported by Travis CI is for actual releases (tags, tarballs, etc.). See http://docs.travis-ci.com/user/deployment/ for what they support, and http://docs.travis-ci.com/user/deployment/releases/ for the only GitHub-related deployment step.